code/game/verbs/who.dm

```diff
@@ -166,25 +151,19 @@
 					num_mentors_online++
 		else
 			for(var/client/C in admins)
-				if(C.holder.rights & R_ADMIN || !(C.holder.rights & (R_MOD|R_MENTOR)))
+				if(C.holder.rights & R_ADMIN || !(C.holder.rights & (R_MENTOR)))
```

Comment:
This is very curious logic. I'd say you could leav this as `if(C.holder.rights & R_ADMIN)`

Reply:
Tbh the only way I touched it is by globally removing R_MOD without reading it, I'll get to the logic sometime later.